bin/eslint-formatters-calypso/README.md

```diff
@@ -0,0 +1,82 @@
+# eslint-formatters-calypso
+
+A set of formatters to be used by ESLint and utilities to build new formatters upon.
+
+Formatters available:
+
+* `lines-modified`: downgrade ESLint `errors` to `warnings`, except for lines modified in the topic branch.
+* `lines-modified-and-specific-rules`: downgrade ESLint `errors` to `warnings`, except for lines modified in the topic branch and the rules in the `.eslines.json` file.
+* `parsing-errors`: takes the ESLint original report and outputs a new one containing only `parsing errors`.
+* `eslines`: a wrapper formatter. If the branch active is `master`, would return the results of `parsing-errors` formatter. Otherwise, would return the results of `lines-modified-and-specific-rules` formatter.
```

Comment:
Few more notes about the `master` functionality:
- Can we somehow devise such `eslines` modes that the app doesn't need to know the term `master`? We have a lot of implicit conditionals based on the branch, which I always forget. Can we move this complexity up somehow?
- If the tool is meant to be reused in other projects, their main branch might not be `master` and the upstream doesn’t need to be `origin`(which is true even for Calypso).